exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Service.java

```diff
@@ -106,6 +106,33 @@ default void resume(BlockchainData blockchainData, byte[] arguments) {
    */
   void createPublicApiHandlers(Node node, Router router);
 
+  /**
+   * An optional callback method invoked by the blockchain before any transactions in a block
+   * are executed.
+   * <!--
+   * FIXME: Or just @see #afterTransactions(BlockchainData) to avoid duplication? Is it acceptable?
```

Comment:
I see a simple link will suffice, as the user won't be forced to read the docs of both methods trying to find the differences between them.

Reply:
Yes, they are so similar it would make sense to document all the peculiarities only once.